src/widgets/index.tsx

```diff
@@ -7,10 +7,18 @@ import {
 } from '@remnote/plugin-sdk';
 import '../style.css';
 import '../App.css';
+import { isMobileOs } from '../lib/utils';
 
 async function onActivate(plugin: ReactRNPlugin) {
   // Register a sidebar widget.
-  await plugin.app.registerWidget('table_of_contents', WidgetLocation.RightSidebar, {
+  let os = await plugin.app.getOperatingSystem();
+
+  let location = WidgetLocation.RightSidebar;
+  if (isMobileOs(os)) {
+    location = WidgetLocation.LeftSidebar
+  }
```

Comment:
[IMO] As same as the above, using const would be more robust.
```suggestion
  const location = isMobileOs(os) ? WidgetLocation.LeftSidebar : WidgetLocation.RightSidebar;
```